resources/lang/it/validation.php

```diff
@@ -22,50 +22,54 @@
     |
     */
 
-    'accepted'   => ':attribute deve essere accettato.',
-    'active_url' => ':attribute non è un URL valido.',
-    'after'      => ':attribute deve essere una data successiva al :date.',
-    'alpha'      => ':attribute può contenere solo lettere.',
-    'alpha_dash' => ':attribute può contenere solo lettere, numeri e trattini.',
-    'alpha_num'  => ':attribute può contenere solo lettere e numeri.',
-    'array'      => ':attribute deve essere un array.',
-    'before'     => ':attribute deve essere una data precedente al :date.',
-    'between'    => [
+    'accepted'             => ':attribute deve essere accettato.',
```

Comment:
Ok, I've used current Laravel indentation for this file: https://raw.githubusercontent.com/laravel/laravel/master/resources/lang/en/validation.php

I reduce it.